apps/nowcasting-app/tailwind.config.js

```diff
@@ -187,7 +187,8 @@ module.exports = {
           600: "#67643B",
           700: "#9F973A",
           800: "#FCED4F",
-          900: "#F19F38"
+          900: "#F19F38",
+          1000: "#595959"
```

Comment:
What's the 1000? Doesn't conform with the usual Tailwind rules so might be slightly confusing to other developers to see, especially as it looks like a mid grey and not a continuation of the delta colours..?

Reply:
the 1000 is the darkest delta color. will update.